moveit_py/README.md

```diff
@@ -0,0 +1,28 @@
+# MoveIt 2 Python Library
+<img src="./banner.png" width="50%">
+
+`moveit_py` is a Python library for interfacing with the core functionalities of Moveit 2.
+The goal of this library is to provide a simplified interface for Moveit 2 Python users.
+
+This Python library depends on [Pybind11](https://pybind11.readthedocs.io/en/stable/index.html) for generating Python bindings.
```

Comment:
I think they spell it pybind11 (all lowercase). Same in the little folder diagram below

Reply:
https://github.com/ros-planning/moveit2/pull/1546/commits/d29f9acd3ff88dfeb93b0d95f43515efebe9c847